test/activities.js

```diff
@@ -0,0 +1,162 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+
+const app = require('../app');
+
+chai.should();
+chai.use(chaiHttp);
+let token;
+
+describe('test activities endpoints', () => {
+  const user = {
+    email: 'bascur@test.com',
+    password: '123456'
+  };
+  before((done) => {
+    chai.request(app)
+      .post('/auth/login')
+      .send(user)
+      .end((err, response) => {
+        response.should.have.status(200);
+        token = response.body.token;
+        done();
+      });
+  });
+  describe('GET /activities', () => {
+    it('get the activities data', (done) => {
+      chai.request(app)
+        .get('/activities')
+        .set({ Authorization: `Bearer ${token}` })
+        .end((err, response) => {
+          response.should.have.status(200);
+          response.should.be.a('object');
```

Comment:
aca deberia decir array,           response.should.be.a('array');

Reply:
Buenas, esperando un array no pasa el test.
<img width="318" alt="array" src="https://user-images.githubusercontent.com/89546054/146693297-522196eb-7d85-41a6-86f1-7fe24cf6afc7.PNG">